src/core/scopes.cc

```diff
@@ -32,6 +32,7 @@
 #include "waflz/limit/rl_obj.h"
 #include "waflz/limit/limit.h"
 #include "waflz/limit/enforcer.h"
+#include "waflz/limit/challenge.h"
```

Comment:
Thinking if we should move challenge.h and challeng.cc out of limits, now that its not part of action for rate-limiting . can do it later too if its getting PR is gettign big.

Reply:
changed